src/lang/en.json

```
@@ -793,13 +793,20 @@
     "noGroupMonitorMsg": "Not Available. Create a Group Monitor First.",
     "Close": "Close",
     "Request Body": "Request Body",
-    "wayToGetFlashDutyKey":"You can go to Channel -> (Select a Channel) -> Integrations -> Add a new integration' page, add a 'Custom Event' to get a push address, copy the Integration Key in the address. For more information, please visit",
-    "FlashDuty Severity":"Severity",
+    "wayToGetFlashDutyKey": "You can go to Channel -> (Select a Channel) -> Integrations -> Add a new integration' page, add a 'Custom Event' to get a push address, copy the Integration Key in the address. For more information, please visit",
+    "FlashDuty Severity": "Severity",
     "nostrRelays": "Nostr relays",
     "nostrRelaysHelp": "One relay URL per line",
     "nostrSender": "Sender Private Key (nsec)",
     "nostrRecipients": "Recipients Public Keys (npub)",
     "nostrRecipientsHelp": "npub format, one per line",
     "showCertificateExpiry": "Show Certificate Expiry",
-    "noOrBadCertificate": "No/Bad Certificate"
-}
+    "noOrBadCertificate": "No/Bad Certificate",
+    "Users": "Users",
+    "Add New User": "Add New User",
+    "confirmDisableUserMsg": "Are you sure you want to disable this user? He will be unable to login.",
```

lets gender this one

```suggestion
    "confirmDisableUserMsg": "Are you sure you want to disable this user? The user will not be able to login anymore.",
```